standalones/MasterBlastR/MasterBlastR.R

```diff
@@ -14,11 +14,11 @@ library(Biostrings)
 
 # USER INPUT -----------------------------------------------------------------------------------
 
-    SampleNo <- "12345"           # Single sample no or "list" for list.csv
+    SampleNo <- "list"           # Single sample no or "list" for list.csv
     Variable <- NA                # Information to be displayed in outputs (MIC value, metadata)
-    LocusID <- "parC"             # Single locus gene or "list" for loci.csv
+    LocusID <- "mtrD"             # Single locus gene or "list" for loci.csv
     curr_work_dir <- "C:\\MasterBlastR\\"
-    ContigsDir <- "C:\\MasterBlastR\\contigs\\"
+    ContigsDir <- "W:\\Projects\\Project_GC_WalterD\\MiSeq\\Gonorrhoea\\contigs\\"
```

Comment:
I'm wondering if instead of including this specific path to the contigs here, could you, when installing and using WADE, make a windows link from `C:\MasterBlastR\contigs` to the actual location where those files are stored? That way, you don't have to change this from the `C:` location?

Alternatively, you could leave the `C:\MasterBlastR\contigs` path in the code on github, and change your local copy to the actual location?